src/biz/APIParser.js

```diff
@@ -16,7 +16,7 @@ export function parseJson(apiReturn) {
     if (comp === 'form') {
       return formGenerator(components.form);
     } else {
-      console.log('api error');
+      console.error('api error');
```

Comment:
This would be a great opportunity to make this error message more meaningful

Reply:
🌽